### ISSUE TITLE
[Feature Request] Option to automatically disable auto-playing of videos

### ISSUE THREAD

**Issue by 32423d** [IsThereAnyDeal/AugmentedSteam#1456]
Every time a user views a store page of a game a promotional video on the store page is automatically played. Clicking the button on the video to not auto-play the video disables auto-playing of videos on store pages until the cache for the browser is cleared in which videos on store pages will start to automatically play again. It would be nice if there was an option given to users in the Augmented Steam settings to automatically disable the auto-playing of videos on store pages so users won't have to manually click a button to disable auto-playing only for that option to get reset after the cache of the web browser is cleared.

**Comment by candela97** [comment]
Yeah last I checked, this setting is stored in cookies so it will be lost if you clear browser cache. This has been requested before, and we can certainly store this preference in the extension, but there are other extensions out there that can enforce this on *all* websites for you, so I'm not sure it's worth duplicating the effort.

**Comment by bottleblue** [comment]
> Yeah last I checked, this setting is stored in cookies so it will be lost if you clear browser cache. This has been requested before, and we can certainly store this preference in the extension, but there are other extensions out there that can enforce this on _all_ websites for you, so I'm not sure it's worth duplicating the effort.

I'd prefer to have the option to disable autoplay within Augmented Steam. I don't need a separate extension to do this and I'm not sure if I'd want to disable it on all websites either. I just know I don't like it when Steam does it.

So please reconsider it.
